### DISCUSSION TITLE
Shop clothing

### DISCUSSION THREAD

**Discussion by Alithegood** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#26] [Q&A] [answered]
C'est normal que les magasins de vêtement ne fonctionne pas ?

**Comment by mafiaa1997** [comment]
Je pense que ca doit-être un peu comme pour les maisons pas encore disponibles pour le moment et viendras plus tard ou alors je me trompe complètement.

**Reply by joelwurtz** [reply, marked as answer]
Cela marche, mais il faut les données associés dans la base de donnée, comme pour les appartements, on le fera mais c'est pas une de nos priorités pour le moment, mais si quelqu'un est motivé pour faire une PR a ce sujet 👍

**Reply by mafiaa1997** [reply]
Merci pour la réponse et je vais essayer de voir si j'arrive à faire quelque chose sur ce sujet
